docker_file/Dockerfile

```diff
@@ -9,15 +9,18 @@ WORKDIR /root/ansible-vsphere-gos-validation
 # Install the required packages
 RUN tdnf makecache && \
     tdnf install -y python3 zip vim wget python3-pip \
-    sshpass gawk nfs-utils iana-etc xorriso && \
+    sshpass gawk nfs-utils iana-etc xorriso python3-lxml && \
```

Comment:
`xorriso` can be removed since Yuhua changed the way to customize autoinstall ISO image.

Reply:
Removed, thanks